test/resources/demo-users/user/managed-ns1/rpa.yaml

```diff
@@ -23,7 +23,7 @@ spec:
       resolver: git
       params:
         - name: url
-          value: "https://github.com/gbenhaim/release-service-catalog.git"
+          value: "https://github.com/konflux-ci/release-service-catalog"
```

Comment:
I don't know if both url formats are supported.

Did you confirm that this repository can be used for this purpose?

Reply:
Yes, I confirmed this with Gal that day but let me check again if this is the right way to replace it

Reply:
The way to confirm this, and the entire pr is to run through the workflow on the Readme and verify it works before the changes and then after the changes.

Reply:
okay will check that

Reply:
Please go through the procedure on the readme as soon as possible and let us know in slack if you run into any issues.

Reply:
I am getting this error while deploying dependencies:
```
klakshma@klakshma-thinkpadp1gen3:~/Documents/github.com/kousalya1998/konflux-ci$ ./deploy-deps.sh
Checking requirements
kubectl is installed
openssl is installed

All requirements are met
Continue
Testing PVC creation for default storage class
Creating PVC from './dependencies/pre-deployment-pvc-binding' using the cluster's default storage class
error: error validating "./dependencies/pre-deployment-pvc-binding": error validating data: failed to download openapi: Get "https://127.0.0.1:38931/openapi/v2?timeout=32s": dial tcp 127.0.0.1:38931: connect: connection refused; if you choose to ignore these errors, turn validation off with --validate=false
```

Reply:
I'm not familiar with this error message, but google suggests it might be some sort of misalignment between the cluster and kubectl client. try updating your oc/kubectl client and kind.

Reply:
When you find the reason for the error and the solution, please create an issues on this github to have it documented.